node_modules/ioslib/lib/certs.js

```diff
@@ -166,7 +166,7 @@ function detect(options, callback) {
 								if (!code) {
 									var tmp = [];
 									parseCerts(out, tmp);
-									results.certs.wwdr = tmp.length && tmp[0].invalid === false;
+									results.certs.wwdr = results.certs.wwdr || (tmp.length && tmp[0].invalid === false);
```

Comment:
I see your point. I'm cool with the fix, but this is the wrong repo. The fix should go in ioslib, not titanium_mobile.

I went ahead and made the fix: https://github.com/appcelerator/ioslib/pull/30.